ENH CMS 6 compatibility (#49)
asyncphp/doorman: README.md

[![CI](https://github.com/silverstripe/doorman/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/doorman/actions/workflows/ci.yml)

Child process management. Needs no extensions.

> The is currently only intended for use with Silverstripe CMS.

Reviewer: ```suggestion
This currently only intended for use with Silverstripe CMS.
```
Small typo - but also I don't know what the purpose of that note is. Was this a fork of something and we had to adopt it? Or does it have any silverstripe cms specific code? If no to both of those, it seems like this is fine for general use?

Author: Removed
